openedx/core/djangoapps/cookie_metadata/__init__.py

```diff
@@ -0,0 +1 @@
+#!/usr/bin/env python3
```

Comment:
Are these shebangs needed for something? (Tests init file has one too.)

Reply:
My editor just automatically adds these to files.

Reply:
I'm gonna remove these.